enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/LdbcAcceptanceTest.scala

```diff
@@ -72,6 +73,34 @@ class LdbcAcceptanceTest extends ExecutionEngineFunSuite with CypherComparisonSu
     all(collectEstimations(result.executionPlanDescription())) should be < 10.0
   }
 
+  test("This LDBC query should work") {
+    // given
+    val ldbcQuery = """MATCH (knownTag:Tag {name:{2}})
+                      |MATCH (person:Person {id:{1}})-[:KNOWS*1..2]-(friend)
+                      |WHERE NOT person=friend
+                      |WITH DISTINCT friend, knownTag
+                      |MATCH (friend)<-[:POST_HAS_CREATOR]-(post)
+                      |WHERE (post)-[:POST_HAS_TAG]->(knownTag)
+                      |WITH post, knownTag
+                      |MATCH (post)-[:POST_HAS_TAG]->(commonTag)
+                      |WHERE NOT commonTag=knownTag
+                      |WITH commonTag, count(post) AS postCount
+                      |RETURN commonTag.name AS tagName, postCount
+                      |ORDER BY postCount DESC, tagName ASC
+                      |LIMIT {3}""".stripMargin
+    eengine.execute(LdbcQueries.Query4.createQuery, LdbcQueries.Query4.createParams)
+
+
+    val params: Map[String, Any] = Map("1" -> 1, "2" ->  "tag1-ᚠさ丵פش", "3" -> 10)
+
+    val result =
+    // when
+      executeWith(Configs.CommunityInterpreted, ldbcQuery, params = params)
```

Comment:
Configs.CommunityInterpreted means that slotted is not supposed to succeed for the query. Is this really what you intended @pontusmelke ?

Reply:
Yes it was picking it up before but failed, now it will use interpreted instead. It is not much work to get it to support `Equals(longslot, refslot)` but I thought it was more prudent to disable in 3.3 and add support for it in 3.4.

Reply:
That sounds reasonable. Otherwise I think it looks fine. Should I forward merge it to 3.4 for now and then you will add the support for ` Equals(longslot,  refslot)`later?

Reply:
yes that will happen as separate work.